### PR TITLE
novacustom: add list of docks and hubs

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

**Comment by macpijan** [comment]
@mkopec Why are we using a bit different structure than in: https://docs.dasharo.com/unified/msi/hcl/ ? What was limiting, or otherwise problematic, to follow it more strictly?

File: docs/unified/novacustom/hw-compatibility.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Hardware compatibility
+
+## Introduction
+
+This page contains the list of peripherals that have been verified to work with
+NovaCustom laptops by the Dasharo validation team. They are tested with each
+release for each laptop currently supported.
```
Comment (macpijan) [inline docs/unified/novacustom/hw-compatibility.md:7]: @BeataZdunczyk @mkopec Let's make sure this is in sync with the latest communication.

Comment (mkopec) [reply docs/unified/novacustom/hw-compatibility.md]: Updated in 93aa9595b91a800d939977382e0f8751f981a280

**Comment by mkopec** [comment]
@macpijan Updated the HCL to follow the MSI HCLs more closely, simplifying language where possible.

**Pull request merged**